keel-core/src/test/kotlin/com/netflix/spinnaker/keel/actuation/IntermittentFailureTests.kt

```diff
@@ -70,6 +71,8 @@ class IntermittentFailureTests : JUnit5Minutests {
     val clock = MutableClock()
     val vetoRepository = mockk<UnhappyVetoRepository>(relaxUnitFun = true)
 
+    val environmentExclusionEnforcer = EnvironmentExclusionEnforcer()
```

Comment:
I'm guessing you'll want to mock responses from this object once the enforcement checks are implemented, so you might want to use mocks in the tests already.